boards/shields/frdm_stbc_agm01/doc/index.rst

```diff
@@ -0,0 +1,71 @@
+.. _frdm_stbc_agm01_shield:
+
+NXP FRDM-STBC-AGM01
+###################
+
+Overview
+********
+
+the FRDM-STBC-AGM01 is an NXP Freedom development board with
+FXOS8700 and FXAS21002. This 9-axis solution (FXAS21002C Gyroscope,
++ FXOS8700CQ E-compass sensor) is compatible with existing Freedom
+development platforms, such as :ref:`frdm_k22f`.
```

Comment:
You can generalize this more and say it's compatible with any board that has an Arduino header